cap_client/cap_api.py

```diff
@@ -268,7 +268,9 @@ def get_field(self, pid, field=None):
                                  })['metadata']
         fields = field.split('.') if field else []
```

Comment:
field has to be specified (dont allow None), then no need to check here

Reply:
I think this should be none for: `cap-client metadata get -p 789ce07b3dbd404daba6fc152
3865eb5` to get just metadata